ccloader/js/mod.js

```diff
@@ -118,7 +129,30 @@ export class Mod {
 	get baseDirectory(){
 		return this._getBaseName(this.file).replace(/\\/g, '/').replace(/\/\//g, '/') + '/';
 	}
-	
+
+	/**
+	 *  Adds all files in folder to path. Optionally limit by file extensions.
+	 * @param {string} relativePath to limit search in
+	 * @param {string[] | undefined} fileExtensions 
+	 * @returns {Promise<boolean>} true if mod is ready and mod is running, otherwise false
+	 */
+	async addAssets(relativePath, fileExtensions) {
```

Comment:
`addAssets` always returns falsish value. Maybe return added assets instead?

Reply:
I'll just return true when it could complete the request successfully. I don't want to return the added assets because the added assets might have already existed in this.manifest.assets.

Reply:
I will revise it anyway.